src/nixblock.cc

```diff
@@ -20,10 +20,10 @@ namespace nixblock {
         sb.set(block.id());
         sb.set(block.type());
         sb.set(block.name());
-        sb.set(block.sourceCount());
-        sb.set(block.dataArrayCount());
-        sb.set(block.tagCount());
-        sb.set(block.multiTagCount());
+        // sb.set(block.sourceCount());
+        // sb.set(block.dataArrayCount());
+        // sb.set(block.tagCount());
+        // sb.set(block.multiTagCount());
```

Comment:
I think it is safe to just remove those completely. Including from the struct builder itself (e.g. `struct_builder sb({ 1 }, { "id", "type", "name", "sourceCount", "dataArrayCount", "tagCount", "multiTagCount" });`